pages/blog/blog-post.js

```diff
@@ -300,6 +300,29 @@ const NavButton = styled(Button)`
   visibility: ${props => (props.href ? 'visible' : 'hidden')};
 `
 
+// Nav buttons
+const NextPostButton = ({ nextPostText, ...rest }) => (
+  <NavButton type="secondary" {...rest} >
+    <span className="longNavText">{nextPostText}</span>
+    <span className="shortNavText">{'>'}</span>
+  </NavButton>
+)
+
+NextPostButton.propTypes = {
+  nextPostText: PropTypes.string.isRequired,
+}
+
+const PreviousPostButton = ({ previousPostText, ...rest }) => (
+  <NavButton type="secondary" {...rest}>
+    <span className="longNavText">{previousPostText}</span>
+    <span className="shortNavText">{'<'}</span>
+  </NavButton>
+)
```

Comment:
This and the NextPostButton is exactly the same code, only the parameter name differs. Could just merge them into one.